### PR TITLE
Different exit codes for errors when running wan2skeaf

### PR DESCRIPTION
**Pull request by npaulish**
301 for the error when parsing the output file, 302 for the missing bxsf, 303 when calculation not finished

### CODE REVIEW

File: aiida_skeaf/calculations/wan2skeaf.py [hunk 1]
```diff
@@ -196,5 +215,5 @@ def get_dict(self) -> dict:
         """Return validated dict."""
         return self.dict
 
-    def __str__(self):
+    def __str__(self):  # pylint: disable=invalid-str-returned
```
Comment (qiaojunfeng) [inline aiida_skeaf/calculations/wan2skeaf.py]: why do you need to override this function?

Comment (npaulish) [reply aiida_skeaf/calculations/wan2skeaf.py]: Hmm, I don't remember changing it, should I just remove the pylint comment and commit again?

Comment (qiaojunfeng) [reply aiida_skeaf/calculations/wan2skeaf.py]: this `__str__` is useless, right? Then just remove it.

Comment (npaulish) [reply aiida_skeaf/calculations/wan2skeaf.py]: should be removed now

**Pull request merged**